models/repo_list_test.go

```diff
@@ -23,7 +23,9 @@ func TestSearchRepositoryByName(t *testing.T) {
 
 	assert.NotNil(t, repos)
 	assert.NoError(t, err)
-	assert.Equal(t, "test_repo_12", repos[0].Name)
+	if assert.NotEmpty(t, repos) {
```

Comment:
No. You should assert how many repos are there.

Reply:
We only need to know that there's more than 0 repos. (since we're accessing `repos[0]`)